python-zict/riscv64.patch

```diff
@@ -0,0 +1,14 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -30,3 +30,11 @@ package() {
+   python setup.py install --prefix=/usr --root="${pkgdir}" --optimize=1 --skip-build
+   install -Dm644 LICENSE.txt -t "${pkgdir}"/usr/share/licenses/${pkgname}/
+ }
++
++source+=(fix-map-size.patch::https://github.com/hack3ric/zict/commit/8aac36794415536b77a1b8318ddffe74278a758f.diff)
```

Comment:
Using `https://github.com/dask/zict/pull/111.diff` can result in a better tracking.

Reply:
The pull request is targeted at 3.0.0 and upward, while the referenced commit is my own backport agains 2.2.0.